tools/profiler/profiler-process.py

```diff
@@ -152,7 +152,7 @@ def postprocess(self):
 
         probe_labels_to_sets = {"group_probe_out": group_active, "se_probe_out": structural_enable_active, "cell_probe_out": cell_enable_active, "primitive_probe_out" : primitive_enable}
 
-        self.trace = {} # cycle number --> set of stacks
+        self.trace = {} # cycle number --> list of stacks
```

Comment:
`{}` looks like an empty dictionary to me

Reply:
The comments are the contents of the dictionary (`key --> value`). I'll update to make this clear!